### PR TITLE
feat: add support for verifying axelard/tofnd binary signatures

### PR DESCRIPTION
**Pull request by talalashraf**

### CODE REVIEW

File: scripts/host.sh [hunk 1]
```diff
@@ -43,13 +44,24 @@ download_dependencies() {
...
         else
             axelard_binary_url="https://axelar-releases.s3.us-east-2.amazonaws.com/axelard/${axelar_core_version}/${axelard_binary}"
+            axelard_binary_signature_url="https://axelar-releases.s3.us-east-2.amazonaws.com/axelard/${axelar_core_version}/${axelard_binary}.asc"
         fi
         curl -s --fail "${axelard_binary_url}" -o "${axelard_binary_path}" && chmod +x "${axelard_binary_path}"
+        if [ -n "$axelard_binary_signature_url" ]; then
+          curl -s --fail "${axelard_binary_signature_url}" -o "${axelard_binary_signature_path}"
+          curl https://keybase.io/axelardev/key.asc | gpg --import
+          printf "\nVerifying Signature of binary. Output: \n================================================"
+          gpg --verify "${axelard_binary_signature_path}" "${axelard_binary_path}"
```
Comment (milapsheth) [inline scripts/host.sh:60]: Is gpg installed by default? We should skip it otherwise and tell validators to install it

Comment (talalashraf) [reply scripts/host.sh:60]: it is on linux and macs.

File: scripts/setup-node.sh [hunk 2]
```diff
@@ -116,6 +116,7 @@ parse_params() {
     logs_directory="$root_directory/logs"
     config_directory="$root_directory/config"
     resources="${git_root}"/resources/"${network}"
+    axelard_binary_signature_path="$bin_directory/axelar-${axelar_core_version}.asc"
```
Comment (RiceAndMeet) [inline scripts/setup-node.sh:119]: I think indentation is off here

File: scripts/setup-validator.sh [hunk 3]
```diff
@@ -127,9 +127,11 @@ parse_params() {
     logs_directory="$root_directory/logs"
     config_directory="$root_directory/config"
     resources="${git_root}"/resources/"${network}"
+    axelard_binary_signature_path="$bin_directory/axelar-${axelar_core_version}.asc"
```
Comment (RiceAndMeet) [inline scripts/setup-validator.sh:130]: indentation here as well ?

File: scripts/setup-node.sh (hunk 2 above)
Comment (RiceAndMeet) [reply scripts/setup-node.sh:119]: nevermind

**Review by RiceAndMeet — APPROVED** [review]

**Pull request merged**